### ISSUE TITLE
Reproduction

### ISSUE THREAD

**Issue by Abcdeabcd** [KaiqiangXiong/CL-MVSNet#9]
Hello author, Thanks for your excellent work!
I have some questions about code reproduction to ask you. I retrained for 16 epochs on a single 48GB GPU without using distributed training. The reproduced results were: acc:0.4411, comp:0.4156, overall:0.4284, and these metrics did not meet expectations. Is this due to training with a single GPU? Or could there be other reasons？
I would greatly appreciate your answers to my questions!

**Comment by KaiqiangXiong** [comment]
Ensuring stable model training is quite critical.  I suggest you check the loss curve.

**Comment by Abcdeabcd** [comment]
Thank you very much for your answer. I will analyze from the perspective of loss curve.
Do you think that training with a single 48GB GPU will affect the training effect?

**Comment by Abcdeabcd** [comment]
Thank you very much for your answer. I will analyze from the perspective of loss curve.Is it possible that interrupting training and then resuming using the 'resume' parameter could lead to unstable convergence?
And Do you think that training with a single 48GB GPU will affect the training effect?

**Comment by CUICHENXI222** [comment]
Hello, while reviewing your code, I noticed that the mask used for calculating Iccloss is a randomly generated filter_mask. Additionally, during the image-level data augmentation process, you not only add Bernoulli noise but also apply the randomly generated filter_mask to the reference image again. This seems inconsistent with the description in the paper. What is the reasoning behind this approach? Below is the relevant source code:

ref_img, filter_mask = random_image_mask(ref_img, filter_size=(ref_img.size(2) // 3, ref_img.size(3) // 3))

**Comment by KaiqiangXiong** [comment]
Based on the experiments, the model occasionally experiences instability during training, such as the loss gradually increasing. In such cases, resuming training from the previous stable checkpoint may resolve the issue. Training with a single GPU tends to get worse results for this training code.
